deploy/runtools/runtime_config.py

```diff
@@ -129,52 +139,26 @@ def array_to_lognames(values, prefix):
 
         command_bootbinaries = array_to_plusargs(all_bootbinaries, "+prog")
         zero_out_dram = "+zero-out-dram" if (enable_zerooutdram) else ""
-        disable_asserts_arg = "+disable-asserts" if (disable_asserts) else ""
+        disable_asserts = "+disable-asserts" if (disable_asserts_arg) else ""
```

Comment:
Renamed this so that the `basecommand` line could be reduced to a `f"..."` string.